java/core/src/main/java/sleeper/core/properties/instance/CommonProperty.java

```diff
@@ -358,7 +358,9 @@ public interface CommonProperty {
     UserDefinedInstanceProperty STATESTORE_COMMITTER_LAMBDA_CONCURRENCY_RESERVED = Index.propertyBuilder("sleeper.statestore.committer.concurrency.reserved")
             .defaultProperty(DEFAULT_LAMBDA_CONCURRENCY_RESERVED)
             .description("The reserved concurrency for the state store committer lambda.\n" +
+                    "Presently this value defaults to 10 to align with expectations around table efficiency. \n" +
```

Comment:
We could explain this is to ensure state store operations can be applied to at least 10 tables even when concurrency is used up in the account.